### PR TITLE
Fix Go modules URLs

### PR DESCRIPTION
**Pull request by lgierth**

### CODE REVIEW

File: static/js/data/bundles_data.js [hunk 1]
```diff
@@ -149,21 +149,21 @@ var data = {
...
-                            'url': 'https://github.com/libp2p/js-libp2p'
+                            'url': 'https://github.com/libp2p/go-ws-transport'
                         }
                     }
                 },
                 "Stream Muxer": {
                     'modules': {
                         'libp2p-yamux': {
                             'status': 'Done',
-                            'url': 'https://github.com/hashicorp/yamux'
```
Comment (jesseclay) [inline static/js/data/bundles_data.js]: Why take this out @lgierth?

Comment (lgierth) [inline static/js/data/bundles_data.js]: It's not a libp2p module -- the yamux related code lives somewhere in go-peerstream. I don't have a strong opinion, was just thinking the link isn't very helpful.

Comment (jesseclay) [inline static/js/data/bundles_data.js]: ok I have 0 opinion here -- @diasdavid? @whyrusleeping?

Comment (Kubuxu) [inline static/js/data/bundles_data.js]: Our code lives here: https://github.com/whyrusleeping/go-smux-yamux

Comment (jesseclay) [inline static/js/data/bundles_data.js]: @lgierth shall we add that link in instead? Or just leave it out?

**Comment by olizilla** [comment]
@diasdavid @lgierth can either of you make a decision? Would be good to get this in or close it out before things move around.

**Comment by lgierth** [comment]
I'm gonna call merge here -- it's better to have missing URLs that we can fill in later, than having wrong or misleading URLs.

**Pull request merged**